consensus/misc/kip71_test.go

```diff
@@ -42,6 +42,35 @@ func TestNextBlockBaseFee(t *testing.T) {
 	}
 }
 
+func TestNextBlockBaseFeeWhenGovernanceUpdated(t *testing.T) {
+	tests := []struct {
+		upperBoundBaseFee uint64 // updated upper bound
+		lowerBoundBaseFee uint64 // updated lower bound
+		parentBaseFee     int64
+		nextBaseFee       int64
+	}{
+		{750000000000, 25000000000, 750000000000, 750000000000},
+		{700000000000, 25000000000, 750000000000, 700000000000},
+		{800000000000, 25000000000, 750000000000, 750000000000},
+		{750000000000, 25000000000, 25000000000, 25000000000},
+		{750000000000, 30000000000, 25000000000, 30000000000},
+		{750000000000, 20000000000, 25000000000, 25000000000},
+	}
+	for i, test := range tests {
+		config := getTestConfig(common.Big2)
+		config.Governance.KIP71.UpperBoundBaseFee = test.upperBoundBaseFee
+		config.Governance.KIP71.LowerBoundBaseFee = test.lowerBoundBaseFee
+		parent := &types.Header{
+			Number:  common.Big3,
+			GasUsed: config.Governance.KIP71.GasTarget, // parentGasUsed == gasTarget
```

Comment:
How about adding the test cases to check the next base fee when `parentGasUsed > gasTarget && parentBaseFee.Cmp(upperBoundBaseFee) == 0` and `parentGasUsed < gasTarget && parentBaseFee.Cmp(lowerBoundBaseFee) == 0`?

Reply:
yes, I missed those cases. thanks.